keras_preprocessing/image/image_data_generator.py

```diff
@@ -543,9 +543,9 @@ def flow_from_dataframe(self, dataframe, directory,
             dataframe: Pandas dataframe containing the filenames
                 (or paths relative to `directory`) of the images in a column and
                 classes in another column/s that can be fed as raw target data.
-            directory: string, path to the target directory that contains all
-                the images mapped in the dataframe.
-                None if x_col column contains absolute paths.
+            directory: Path to the directory to read images from. Directory to
```

Comment:
Please follow the code convention

`arg_name: type, description`

Reply:
done